launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -147,15 +148,24 @@ def _load_nodes_in_docker(
                 executable=executable_name
             )
 
-            self._container.exec_run(
+            log_generator = self._container.exec_run(
                 cmd=cmd,
-                detach=True,
                 tty=True,
+                stream=True,
             )
 
+            context.asyncio_loop.run_in_executor(self._executor, self._handle_logs, log_generator)
+
             self.__logger.debug('Running \"{}\" in container: \"{}\"'
                                 .format(cmd, self._policy.container_name))
 
+    def _handle_logs(
+        self,
+        log_generator
```

Comment:
Could we get the typing for this too?

Reply:
I actually don't know the type of `log_generator`. Its an untyped variable returned by docker-py API.

Reply:
Docstring for function then?

Reply:
I can do that